src/test/loader.ts

```diff
@@ -98,7 +98,7 @@ export class Loader {
     this._fullConfig.shard = takeFirst(this._configOverrides.shard, this._config.shard, baseFullConfig.shard);
     this._fullConfig.updateSnapshots = takeFirst(this._configOverrides.updateSnapshots, this._config.updateSnapshots, baseFullConfig.updateSnapshots);
     this._fullConfig.workers = takeFirst(this._configOverrides.workers, this._config.workers, baseFullConfig.workers);
-    this._fullConfig.webServer = takeFirst(this._configOverrides.webServer, this._config.webServer, baseFullConfig.webServer);
+    this._fullConfig.launch = takeFirst(this._configOverrides.launch, this._config.launch, baseFullConfig.launch);
```

Comment:
You want to normalize right here, see `toReporters` above that ensures an array.

Reply:
cool!